persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -60,12 +61,16 @@ class HmdaFileParser(submissionId: SubmissionId) extends HmdaPersistentActor wit
 
     case ReadHmdaRawFile(persistenceId) =>
       publishEvent(ParsingStarted(submissionId))
+      var encounteredParsingErrors: Boolean = false
```

Comment:
It's more for convention than anything else, most Akka actor code you will see will follow that pattern. The idea is it's easy to spot where the state changes will happen, and there will be an initial state at actor instantiation before any messages are received (this could be necessary in some cases; I think it doesn't necessarily matter in this particular case)